src/vehicle.cpp

```diff
@@ -521,7 +528,16 @@ void vehicle::init_state(int init_veh_fuel, int init_veh_status)
             }
         }
     }
-
+    // destroy tires until the vehicle is not drivable
+    if( destroyTires ) {
```

Comment:
Needs a check for `wheelcache.empty()`, otherwise it will crash if a wheel-less vehicle is generated.